docs/usage/SHOPPING_LIST_SYNC.md

```diff
@@ -0,0 +1,190 @@
+# Shopping List Sync
+
+Module used for importing/syncing Shopping Lists into a commercetools project. 
+It also provides utilities for generating update actions based on the comparison of a [ShoppingList](https://docs.commercetools.com/api/projects/shoppingLists#shoppinglist) 
+against a [ShoppingListDraft](https://docs.commercetools.com/api/projects/shoppingLists#shoppinglistdraft).
+
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Usage](#usage)
+  - [Sync list of Shopping List Drafts](#sync-list-of-shopping-list-drafts)
+    - [Prerequisites](#prerequisites)
+    - [About SyncOptions](#about-syncoptions)
+    - [Running the sync](#running-the-sync)
+    - [More examples of how to use the sync](#more-examples-of-how-to-use-the-sync)
+  - [Build all update actions](#build-all-update-actions)
+  - [Build particular update action(s)](#build-particular-update-actions)
+- [Caveats](#caveats)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+## Usage
+
+### Sync list of Shopping List Drafts
+
+#### Prerequisites
+1. Create a `sphereClient`:
+Use the [ClientConfigurationUtils](https://github.com/commercetools/commercetools-sync-java/blob/2.3.0/src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java#L45) which apply the best practices for `SphereClient` creation.
+If you have custom requirements for the sphere client creation, have a look into the [Important Usage Tips](IMPORTANT_USAGE_TIPS.md).
+
+2. The sync expects a list of `ShoppingList`s that have their `key` fields set to be matched with shopping lists in the 
+target CTP project. The customers in the target project need to have the `key` fields set, otherwise they won't be 
+matched.
+
+3. Every shopping list may have a reference to their [Type](https://docs.commercetools.com/api/projects/shoppingLists#set-custom-type) of their custom fields. 
+The `Type` references should be expanded with a key.
+Any reference that is not expanded will have its id in place and not replaced by the key will be considered as existing 
+resources on the target commercetools project and the library will issue an update/create an API request without reference
+resolution.
+
+     - When syncing from a source commercetools project, you can use [`mapToShoppingListDraft`](https://commercetools.github.io/commercetools-sync-java/v/2.4.0/com/commercetools/sync/shoppinglists/utils/ShoppingListReferenceResolutionUtils.html#mapToShoppingListDrafts-java.util.List-)
```

Comment:
This link doesn't work. I guess it will be added later?

Reply:
Yes, it will work once the release is done and release version is 2.4.0